docs/src/main/paradox/file.md

```diff
@@ -68,6 +68,29 @@ Scala
 Java
 : @@snip (../../../../file/src/test/java/akka/stream/alpakka/file/javadsl/DirectoryChangesSourceTest.java) { #minimal-sample }
 
+### LogRotationSink
+
+The `LogRotationSink` will create and write to multiple files.  
+This sink will wait for a function as parameter which returns a
```

Comment:
`takes` or `accepts`, the sink will not exist before creating an instance so it doesn't really wait for anything.